projects/Inventaire/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = ''
+      Inventaire is a libre/free webapp to make inventories and lists of books, and facilitates book sharing. By
+      aggregating individuals' and collectives' book inventories from around the world, Inventaire is kind of a
+      huge, distributed community library.
+    '';
+    subgrants = [
+      "Inventaire"
+      "Inventaire2"
+      "Inventaire-Self-hosted"
+    ];
+  };
+
+  nixos = {
+    modules.services.inventaire = {
+      module = ./module.nix;
+      examples = {
+        basic = {
+          description = ''
+            A (very insecure!) example of setting up Inventaire and its dependencies on the local machine.
+          '';
+          module = ./examples/basic.nix;
+          tests.basic = import ./tests/basic.nix args;
```

The [test data type](https://github.com/ngi-nix/ngipkgs/blob/0550976adba8fe48e8bf39d62e4cf143ac3f83ed/projects/types.nix#L218-L232) has been changed and expects this to be stored in a `module` option, now, so this will fail when we rebase.

```suggestion
          tests.basic.module = import ./tests/basic.nix args;
```